cva6/sim/Makefile

```diff
@@ -37,6 +39,40 @@ isspostrun_opts ?=
 log             ?=
 variant         ?=
 
+# TRACE_FAST, TRACE_COMPACT and VERDI are mutually exclusive and imply non-empty DEBUG.
+ifneq ($(TRACE_FAST),)
+ifneq ($(TRACE_COMPACT),)
+$(error  ariables TRACE_FAST and TRACE_COMPACT are mutually exclusive, please unset one of them)
```

Comment:
s/ ariables/Variables/

Reply:
Addressed in PR #1427 which supersedes the present one (#1424).